phd.tex

```diff
@@ -285,6 +285,8 @@
 
 \cleardoublepage
 \tableofcontents
+% Somehow the clear double page, newpage, and pagebreak below here result in the intro chapter on the very next page
+% From the table of contents
```

Comment:
Don't know if it is intentional, but the intro chapter doesn't have a blank page before it while the source seems to indicate that that was an intention

Reply:
had to use this hack: https://tex.stackexchange.com/questions/36880/insert-a-blank-page-after-current-page